app/src/main/java/com/zeapo/pwdstore/autofill/oreo/AutofillScenario.kt

```diff
@@ -183,14 +191,15 @@ fun Dataset.Builder.fillWith(
 ) {
     val credentialsToFill = credentials ?: Credentials(
         "USERNAME",
-        "PASSWORD"
+        "PASSWORD",
+        "OTP"
     )
     for (field in scenario.fieldsToFillOn(action)) {
-        val value = if (field == scenario.username) {
-            credentialsToFill.username
-        } else {
-            credentialsToFill.password
-        } ?: continue
```

Comment:
Note: The `?: continue` had to go because we may now be in a situation where we generate an Autofill dataset without any non-null value (e.g. if the decrypted entry does not have an OTP secret). But we have to call `setValue` at least once or the `DatasetBuilder` will crash.